src/create/components/Create.tsx

```diff
@@ -0,0 +1,207 @@
+import Tab from "./Tab";
+
+export type TabType = "전체" | "진행 중인 일정" | "완료된 일정" | "임시 저장";
+
+interface DateProps {
+  start: Date | undefined;
+  end: Date | undefined;
+}
+
+interface CardItemType {
+  theme: string;
+  img?: string;
+  title: string;
+  writer: string;
+  status: boolean;
+  content?: string;
+  durationStart: string;
+  durationEnd: string;
+}
+
+const Create = () => {
+  const [tab, setTab] = useState<TabType>("전체");
+  const [CardList, setCardList] = useState<CardItemType[]>(defaultCardList);
+
+  const [title, setTitle] = useState("");
+  const [date, setDate] = useState<DateProps>({
+    start: undefined,
+    end: undefined,
+  });
+  const [showCalendar, setShowCalendar] = useState({
+    start: false,
+    end: false,
+  });
+
+  const handleCalendarClick = (type: "start" | "end") => {
+    setShowCalendar((prev) => ({
+      start: type === "start" ? !prev.start : false,
+      end: type === "end" ? !prev.end : false,
+    }));
+  };
+
+  const handleStartDateChange = (newDate: Date) => {
+    setDate((prevDate) => ({
+      ...prevDate,
+      start: newDate,
+    }));
+    handleCalendarClick("start");
+  };
+
+  const handleEndDateChange = (newDate: Date) => {
+    setDate((prevDate) => ({
+      ...prevDate,
+      end: newDate,
+    }));
+    handleCalendarClick("end");
+  };
+
+  const onClickTab = (tab: TabType) => {
+    setTab(tab);
+    setShowCalendar({
+      start: false,
+      end: false,
+    });
+  };
+
+  // TODO: 삭제 요청 추가
+  const onClickDelete = (i: number) => {
+    const updatedCardList = [...CardList];
+    updatedCardList.splice(i, 1);
+    setCardList(updatedCardList);
+  };
+
+  // tab으로 필터링
+  const filteredInTab = (tab: string, dataList: CardItemType[]) => {
+    const currentDate = new Date();
+
+    if (tab === "임시 저장") {
+      return dataList.filter((item) => !item.status);
+    } else if (tab === "진행 중인 일정") {
+      return dataList.filter((item) => {
+        const cardStartDate = new Date(item.durationStart);
+        const cardEndDate = new Date(item.durationEnd);
+        return (
+          !item.status &&
+          cardStartDate <= currentDate &&
+          cardEndDate >= currentDate
+        );
+      });
+    } else if (tab === "완료된 일정") {
+      return dataList.filter((item) => {
+        const cardEndDate = new Date(item.durationEnd);
+        return !item.status && cardEndDate < currentDate;
+      });
+    } else {
+      return dataList;
+    }
+  };
+
+  // date, tab, title 이 바뀔 때마다 필터링 진행
+  useEffect(() => {
+    const filteredList = filteredInTab(tab, defaultCardList);
+
+    const filteredCards = filteredList.filter((card) => {
+      const cardStartDate = new Date(card.durationStart);
+      const cardEndDate = new Date(card.durationEnd);
+
+      if (date.start && date.end) {
+        return (
+          cardStartDate >= date.start &&
+          cardEndDate <= date.end &&
+          card.title.toLowerCase().includes(title.toLowerCase())
```

Comment:
```suggestion
          
      if (!card.title.toLowerCase().includes(title.toLowerCase())) return false;
      
```
중복되는 조건은 최상단에 걸어주는게 좋습니다!
최상단에 해당 조건을 걸어주면 최하단 [아무날짜도 선택되지않은 경우] return true만 해주고, 나머지 코드에 해당 조건을 제외시킬수있습니다!

Reply:
확인했습니당 :)